README.md

```diff
@@ -1 +1,107 @@
-# appointment-diary-front-end
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)

```

Comment:
There is no pull request for the API project, so I'll leave my suggestions here:

- [ ] You are using Rspec, but the `Test` folder is used by another different test framework.  Please, kindly consider deleting that folder if you are not using it.
- [ ] 27 tests didn't work for me.  Please, check if those tests work in your environment and if they don't, kindly fix them to ensure your app's integrity.
- [ ] On your readme, it would be nice to add a screenshot.  It could be your models' diagram or even your API docs.  That way, your repo will look more professional.
- [ ] There is no license file on your API project.  Please, grab the license file from [here](https://github.com/microverseinc/readme-template), and include it on the root of your project.  After that, please update the `license `section on your readme to include a working link to that file.
- [ ] For your next review request, kindly please open a pull request on your API project, with all the changes `dev => main`.  That way, it will be easier to deliver to you a clearer review.

Reply:
@Meltrust  All suggested changes implemented 👍🏽